compiler/compiler/src/main/java/org/robovm/compiler/config/Config.java

```diff
@@ -206,6 +208,15 @@
     private boolean manuallyPreparedForLaunch = false;
     private int threads = Runtime.getRuntime().availableProcessors();
     private Logger logger = Logger.NULL_LOGGER;
+    
+  //fields used for global program optimisation
+  	private boolean useGlobalOpt = false;
+  	private boolean skipInlining = false;
+  	private boolean skipDevirtualization = false;
+
+  	private float   inlineExpansionFactor = 3.0f;
```

Comment:
What factor?
Which size?
Size of bytecode instructions?
Size of native code?

Reply:
I tooked the same parameters as the soot framework does
See 
https://www.sable.mcgill.ca/soot/tutorial/phase/index.html under the StaticInliner section to find more about the numbers are taken

Reply:
the size in fact is in term of the number of units in a jimple method